model/http-common.yaml

```diff
@@ -14,7 +14,8 @@ groups:
         examples: ['timeout', 'name_resolution_error', '500']
         note: |
           If the request fails with an error before response status code was sent or received,
-          `error.type` SHOULD be set to exception type or a component-specific low cardinality error code.
+          `error.type` SHOULD be set to exception type (its fully-qualified class name, if applicable)
+          or a component-specific low cardinality error code.
```

Comment:
Nit: Instead of error "code", error "identifier" may be better. The examples list for example "name_resolution_error" which to me looks neither like a numeric code nor an exception class name (much less a fully qualified one)